pkg/datarepair/checker/config.go

```diff
@@ -18,7 +24,18 @@ type Config struct {
 
 // Initialize a Checker struct
 func (c Config) initialize(ctx context.Context) (Checker, error) {
-	return &checker{}, nil
+	pointerdb := pointerdb.LoadFromContext(ctx)
+	overlay := overlay.LoadServerFromContext(ctx)
+	client, err := redis.NewClientFrom(c.QueueAddress)
+	if err != nil {
+		return nil, Error.Wrap(err)
+	}
+	repairQueue := queue.NewQueue(client)
+	check := newChecker(pointerdb, repairQueue, overlay, 0, zap.L())
```

Comment:
I think we shouldn't have to do this setup after calling new checker. Could we not do this in new checker itself ?